devices.js

```diff
@@ -4713,6 +4713,66 @@ const devices = [
         fromZigbee: [fz.on_off],
         toZigbee: [tz.on_off],
     },
+    {
+        zigbeeModel: ['RM01'],
+        model: 'RM01',
+        vendor: 'Busch-Jaeger',
+        description: 'Busch-Jaeger 6735, 6736, 6737',
+        supports: 'on/off',
+        endpoint: (device) => {
+            return {'row_1': 0x0a, 'row_2': 0x0b, 'row_3': 0x0c, 'row_4': 0x0d, 'light': 0x12};
```

Comment:
I'm a bit hesitate to add `light` here as any of the postfixes defined in https://github.com/Koenkk/zigbee2mqtt/pull/2723/files#diff-13ce215eb47faacd9d4566b0f1212d00R33 cannot be used in a `friendly_name` anymore and I think `light` is quite a common used one. (e.g. `zigbee2mqtt/kitchen/light`). 

Can you try changing `light` to `default`, you should be able to control it via `zigbee2mqtt/[FRIENDLY_NAME]/set`.

Reply:
I'll happily change this to whatever you suggest - except for `default`, which seems to have a special meaning for zigbee2mqtt. Since endpoint 0x12 is only available with two of the three devices behind the interface - the dimmer & the relais - using `default` will result in all kinds of exceptions because zigbee2mqtt cannot find the default endpoint when using the power adapter as back-end.

Reply:
Okay, renamed light to relay - does that sound better to you?

Reply:
Yes perfect